Foundation.hs

```diff
@@ -147,10 +152,22 @@ instance YesodAuth App where
                 }
 
     -- You can add other plugins like Google Email, email or OAuth here
-    authPlugins _ = [authOpenId Claimed []]
+    authPlugins app = [authOpenId Claimed []] ++ extraAuthPlugins
+      -- Enable authDummy login when in development mode.
+      where extraAuthPlugins = [authDummy | appDevelopment $ appSettings app]
+
+
 
     authHttpManager = getHttpManager
 
+-- | Access function to determine if a user is logged in.
+isAuthenticated :: Handler AuthResult
+isAuthenticated = do
+    mu <- maybeAuthId
```

Comment:
I think `mu` is a little unclear, and not what people usually seem to use based on searching Github: https://github.com/search?l=haskell&q=maybeAuthId&type=Code&utf8=%E2%9C%93